src/ThreadPool/ThreadPool/MyThreadPool.cs

```diff
@@ -0,0 +1,177 @@
+﻿namespace ThreadPool;
+
+using System.Collections.Concurrent;
+
+/// <summary>
+/// Custom threadpool for parallel task executing
+/// </summary>
+public class MyThreadPool
+{
+    private static readonly ConcurrentQueue<Action> TasksQueued = new();
```

Comment:
```suggestion
    private static readonly ConcurrentQueue<Action> _tasksQueued = new();
```
Это же по сути обычное поле